src/bidiMapper/bidiMapper.ts

```diff
@@ -26,6 +26,7 @@ export type {
   ICdpConnection as CdpConnection,
 } from '../cdp/cdpConnection.js';
 export type {ICdpClient, ICdpClient as CdpClient} from '../cdp/cdpClient.js';
+export {EventEmitter} from '../utils/EventEmitter.js';
```

Comment:
This is needed as Puppeteer EventEmitter is incomparable with BiDi Mapper EventEmitter.

Reply:
Can this be `export type`, though?

Reply:
Types are different, I am happy if we align with something external - Node EventEmitter and we use that as a CommonInterface. 
Else we can't as you extend the interface CDPClient with the class and that brakes Typescript as its looking for `#private` which can't be provided.

Reply:
Sure, we could look into that in another PR.